processing/src/main/java/org/apache/druid/query/aggregation/first/StringFirstLastUtils.java

```diff
@@ -36,11 +36,16 @@
   private static final int NULL_VALUE = -1;
 
   /**
-   * Returns whether a given value selector *might* contain SerializablePairLongString objects.
```

Comment:
Yes, `selectorNeedsFoldCheck` method in `StringFirstLastUtils` is now shared by long/float/doubleFirst/Last, it should be extracted out of this class. 

 I have not made more changes to this class file because stringFirst/Last will be refactored in a new PR to share new abstract classes provided in this PR, which means this class is also involved.

Reply:
oh ok. so you will make the changes in that PR. is that right?

Reply:
Yes